### PR TITLE
Add `translatable_content` to translation types.

### PR DESCRIPTION
**Pull request by zedzior**
I want to merge this change, because I want to see in translation subscription types, what item was translated. It adds `translatable_content`, including all translatable fields and ID of translated object, to all translation types.

**API changes:** (category example)
```graphql
class CategoryTranslation(BaseTranslationType):
   ...
   translatableContent: CategoryTranslatableContent
      ...
      categoryId: ID!
```  

https://linear.app/saleor/issue/MERX-239/buglack-of-id-of-translated-entity-in-user-defined-translation-webhook

# Impact

- [ ] New migrations
- [x] New/Updated API fields or mutations
- [ ] Deprecated API fields or mutations
- [ ] Removed API types, fields, or mutations

# Docs

- [ ] Link to documentation:

# Pull Request Checklist

- [x] Privileged queries and mutations are either absent or guarded by proper permission checks
- [x] Database queries are optimized and the number of queries is constant
- [ ] Database migrations are either absent or optimized for zero downtime
- [x] The changes are covered by test cases
- [x] All new fields/inputs/mutations have proper labels added (`ADDED_IN_X`, `PREVIEW_FEATURE`, etc.)
- [ ] All migrations have proper dependencies
- [ ] All indexes are added concurrently in migrations
- [ ] All RunSql and RunPython migrations have revert option defined

### CODE REVIEW

**Review by IKarbowiak — APPROVED** [review]

**Review by yellowee — APPROVED** [review]

**Review by korycins — APPROVED** [review]

**Review by kadewu — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ All notable, unreleased changes to this project will be documented in this file.
 ### Breaking changes
 
 ### GraphQL API
+- Add `translatableContent` to all translation types - #15617 by @zedzior
+- Add translated object id to all translatable content types - #15617 by @zedzior
```
Comment (IKarbowiak) [inline CHANGELOG.md:13]: Could be merged into one

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/translations/types.py [hunk 2]
```diff
@@ -100,27 +113,51 @@ class AttributeValueTranslation(
...
+        return AttributesByAttributeId(info.context).load(root.attribute_id)
+
 
 class AttributeTranslatableContent(ModelObjectType[attribute_models.Attribute]):
-    id = graphene.GlobalID(required=True, description="The ID of the attribute.")
+    id = graphene.GlobalID(
+        required=True, description="The ID of the attribute translatable content."
+    )
+    attribute_id = graphene.ID(
+        required=True, description="The ID of the attribute to translate."
```
Comment (korycins) [inline saleor/graphql/translations/types.py:159]: Don't you need ADDED_IN here?

File: saleor/graphql/translations/types.py [hunk 3]
```diff
@@ -145,13 +182,20 @@ class Meta:
...
+
 
 class AttributeValueTranslatableContent(
     ModelObjectType[attribute_models.AttributeValue]
 ):
     id = graphene.GlobalID(
         required=True, description="The ID of the attribute value translatable content."
     )
+    attribute_value_id = graphene.ID(
+        required=True, description="The ID of the attribute value to translate."
```
Comment (korycins) [inline saleor/graphql/translations/types.py:197]: ADDED_IN?

File: saleor/graphql/translations/types.py [hunk 4]
```diff
@@ -199,17 +247,31 @@ class ProductVariantTranslation(
...
+    ):
+        return ProductVariantByIdLoader(info.context).load(root.product_variant_id)
+
 
 class ProductVariantTranslatableContent(ModelObjectType[product_models.ProductVariant]):
     id = graphene.GlobalID(
         required=True, description="The ID of the product variant translatable content."
     )
+    product_variant_id = graphene.ID(
+        required=True, description="The ID of the product variant to translate."
```
Comment (korycins) [inline saleor/graphql/translations/types.py:273]: ADDED_IN?

**Review by korycins — APPROVED** [review]

**Pull request merged**